src/components/common/Select.js

```diff
@@ -51,11 +50,10 @@ const OptionsWrapper = styled.div`
   `}
 `;
 
-const Select = ({ options = [], onChange, value }) => {
+export default function Select({ options = [], onChange, value }) {
```

Comment:
Why convert this from arrow function?

Reply:
So I can do the export default in the line I define the component, and also, I wasn't using the implicit return of the arrow function, so felt like kinda pointless to use it

Reply:
So you won't be using proptypes?

Reply:
Yeah, I'm using propTypes, export default doesn't have to be the last line in a module